iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/RemoveHvdcLineBuilder.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.modification.topology;
+
+import java.util.List;
+
+/**
+ * @author Anis Touri <anis-1.touri@rte-france.com>
+ */
+
+public class RemoveHvdcLineBuilder {
+    private String hvdcLineId;
+    private List<String> shuntCompensatorIds;
+
+    public RemoveHvdcLine build() {
+        return new RemoveHvdcLine(hvdcLineId, shuntCompensatorIds);
```

Comment:
Before, building the `RemoveHvdcLine` you have to check that parameters of the builder have been correctly checked. So here, hvdcLineId should be not null